ui_framework/components/menu/menu.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from 'enzyme';
+import { requiredProps } from '../../test/required_props';
+
+import {
+  KuiMenu,
+} from './menu';
+
+test('renders KuiMenu', () => {
+  const component = <KuiMenu { ...requiredProps }>children</KuiMenu>;
+  expect(render(component)).toMatchSnapshot();
+});
```

Comment:
Can we tweak the format of these descriptions and also add a test for the `contained` prop?

```javascript
test('baseline', () => {
  const component = <KuiMenu { ...requiredProps }>children</KuiMenu>;
  expect(render(component)).toMatchSnapshot();
});

test('contained prop', () => {
  const component = <KuiMenu contained>children</KuiMenu>;
  expect(render(component)).toMatchSnapshot();
});
```